pkg/operator/ceph/client/controller.go

```diff
@@ -268,6 +303,16 @@ func (r *ReconcileCephClient) createOrUpdateClient(cephClient *cephv1.CephClient
 		}
 	}
 
+	if shouldRotateCephxKeys {
+		// rotate the CephX key if the user requested it
+		rotatedKey, err := r.rotateCephXKeys(cephClient, clientEntity)
```

Comment:
I don't think a function is necessary or helpful for 2 lines of code and a log message.

Also, because `keyring.SecretStore` isn't being used here today, it woudld be more straightforward to use the `cephclient.AuthRotate()` method, which uses the same abstraction layer as `cephclient.AuthGetOrCreateKey()` and `cephclient.AuthUpdateCaps()` above. That will reduce the lines of code to 1, further making a helper function unnecessary.